modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/servlet/taglib/util/UserActionDropdownItems.java

```diff
@@ -95,7 +98,9 @@ public List<DropdownItem> getActionDropdownItems() {
 					_getPermissionsActionUnsafeConsumer()
 				).add(
 					() ->
-						(PropsValues.LAYOUT_USER_PRIVATE_LAYOUTS_ENABLED ||
+						(PrefsPropsUtil.getBoolean(
+							CompanyThreadLocal.getCompanyId(),
```

Comment:
@torresvictor100, get the `companyId` following the rule below:

- If the `themeDisplay` is available on the method or class being changed then use `themeDisplay.getCompanyId()`
- If the `portletRequest` or the `httpServletRequest` is available on the method or class being changed then use `PortalUtil.getCompanyId(portletRequest)` or `PortalUtil.getCompanyId(httpServletRequest)`
- Otherwise, keep using the method `CompanyThreadLocal.getCompanyId()`

Lastly, visit all commits like this one to apply this rule. :wink: